samtranslator/internal/schema_source/aws_serverless_graphqlapi.py

```diff
@@ -13,7 +19,7 @@ class Auth(BaseModel):
 
 
 class Logging(BaseModel):
-    CloudWatchLogsRoleArn: Optional[str]
+    CloudWatchLogsRoleArn: Optional[SamIntrinsicable[str]]
```

Comment:
@hoffa @aaythapa Are we still supposed to be marking fields as intrinsicable on SAM models?

Reply:
Sure they could be marked as passthrough then according to that definition. I was marking properties as `PassThrough` if there was no interaction with the value in the transform code outside of just assigning it to the CFN resource

Reply:
Updated to PassThroughProp.